src/pybind/mgr/telemetry/module.py

```diff
@@ -864,14 +919,14 @@ def gather_device_report(self) -> Dict[str, Dict[str, Dict[str, str]]]:
                 m = self.remote('devicehealth', 'get_recent_device_metrics',
                                 devid, min_sample)
             except Exception as e:
-                self.log.debug('Unable to get recent metrics from device with id "{}": {}'.format(devid, e))
+                self.log.error('Unable to get recent metrics from device with id "{}": {}'.format(devid, e))
```

Comment:
@ljflores the `self.log.exception` fixes were lost in the force push in the main PR and here too; not a big issue, can be fixed later

Reply:
Making a note to self to fix this in another PR.